backend/oasst_backend/user_repository.py

```diff
@@ -119,6 +120,12 @@ def mark_user_deleted(self, id: UUID) -> None:
 
         user.deleted = True
 
+        # Anonymise user data
+        user.display_name = "Deleted User"
```

Comment:
I see the intention is it to keep this info coherent for inference and backend user data.
like it is used in inference/server/oasst_inference_server/routes/admin.py as well.

Can this be defined in oasst-shared/utils.py.  ?
example
` DELETED_USER = "Deleted User"` 
Similar changes can be thought for the hardcoded prefix "deleted_"
` user.username = f"deleted_{user.id}`